artemis-cdi-client/src/main/java/org/apache/artemis/client/cdi/extension/ArtemisClientConfigBean.java

```diff
@@ -45,7 +45,8 @@ public Set<InjectionPoint> getInjectionPoints() {
       return emptySet();
    }
 
-   @Override
+   // we cannot use @Override here because otherwise the build of artemis-cdi-jakarta-client fails since the method was removed in jakarta ee 9+
```

Comment:
Still need to update the module name following its rename.

(same in other class changed)